pkg/provisioners/utils/topic_names.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+import (
+	"fmt"
+)
+
+func TopicName(channelNamespace, channelName string) string {
+	return fmt.Sprintf("knative-eventing-channel_%s.%s", channelNamespace, channelName)
```

Comment:
Hmmm, this might not be general enough, as I'm sure we can find some Channels that won't allow underscores or dots. Maybe have the separator as another argument?